packages/theme/src/pageTheme.ts

```diff
@@ -63,12 +64,25 @@ export const colorVariants: Record<string, string[]> = {
  * As the background shapes and colors are decorative, we place them onto the
  * page as a css background-image instead of an html element of its own.
  */
-export function genPageTheme(colors: string[], shape: string): PageTheme {
-  const gradientColors = colors.length === 1 ? [colors[0], colors[0]] : colors;
+export function genPageTheme(options: {
+  colors: string[];
+  shape: string;
+  fontColor?: string;
```

Comment:
Let's make the 3rd argument an optional options object, just in case we want to add more stuff in the future

Reply:
Sounds good, done 👍